packages/cli/src/middleware/index.js

```diff
@@ -1,60 +1,24 @@
-import { getTruffleConfig } from '../helpers/truffle-config'
-import {
-  loadManifestFile,
-  loadArappFile,
-} from '../lib/environment/loadConfigFiles'
-import {
-  configEnvironment,
-  NoEnvironmentInArapp,
-  NoEnvironmentInDefaults,
-  NoNetworkInTruffleConfig,
-} from '../lib/environment/configEnvironment'
+import { loadArappFile } from '@aragon/toolkit'
 
 class InvalidArguments extends Error {}
 
-export function configCliMiddleware(argv) {
-  const [cmd, subcmd] = argv._
-
+export function configEnvironmentMiddleware(argv) {
```

Comment:
The only transformation that we do on the middleware is parsing the global options `--use-frame` and `--environment` to return a modified environment with: `frame:<environment>` if the `use-frame` was enable.